File: 1сем/ОАиП лабы/Common7/IDE/ProjectTemplatesCache/VisualBasic/WCF/1049/StateMachineWorkflowServiceLibrary/IWorkflow1.vb
' ПРИМЕЧАНИЕ. Можно использовать команду "Переименовать" в контекстном меню, чтобы изменить имя интерфейса "IWorkflow1" в коде и файле конфигурации.
<ServiceContract()> _
Public Interface IWorkflow1

    <OperationContract()> _
    Function GetData(ByVal value As Integer) As String

    ' TODO: Добавьте здесь операции служб

End Interface

File: 1сем/ОАиП лабы/Common7/IDE/ProjectTemplates/CSharp/WCF/1049/StateMachineWorkflowServiceLibrary/IWorkflow1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ServiceModel;

namespace $safeprojectname$
{
	// ПРИМЕЧАНИЕ. Команду "Переименовать" в меню "Рефакторинг" можно использовать для одновременного изменения имени интерфейса "IWorkflow1" в коде и файле конфигурации.
	[ServiceContract]
	public interface IWorkflow1
	{

		[OperationContract]
		string GetData(int value);

		// TODO: Добавьте здесь операции служб
	}
}
